noir/noir-repo/aztec_macros/src/transforms/compute_note_hash_and_nullifier.rs

```diff
@@ -100,7 +107,7 @@ fn generate_compute_note_hash_and_nullifier_source(note_types: &[String]) -> Str
         // so we include a dummy version.
         "
         unconstrained fn compute_note_hash_and_nullifier(
-            contract_address: AztecAddress,
+            contract_address: dep::aztec::protocol_types::address::AztecAddress,
```

Comment:
nice find!

Reply:
Actually @just-mitch run into this on Thursday 😢  One of the consequences of moving this generation into the interner is also really poor error messages, but I have an idea on how to improve them